infrastructure/scripts/conf.sh

```diff
@@ -2,34 +2,20 @@
 set -eo pipefail
 IFS=$'\n\t'
 
-source /vagrant/scripts/lib/k8s.sh
-if [ -z ${AIRY_VERSION+x} ]; then
-    branch_name="$(git symbolic-ref HEAD 2>/dev/null)" ||
-    branch_name="(unnamed branch)"     # detached HEAD
-
-    branch_name=${branch_name##refs/heads/}
-    case "$branch_name" in
-        develop )
-            AIRY_VERSION=beta
-            ;;
-        release* )
-            AIRY_VERSION=release
-            ;;
-        * )
-            AIRY_VERSION=latest
-            ;;
-    esac
+SCRIPT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+INFRASTRUCTURE_PATH=$( cd ${SCRIPT_PATH}/../; pwd -P)
+
+if [[ ! -f ${INFRASTRUCTURE_PATH}/airy.conf ]]; then
```

Comment:
Don't run the provisioning if `airy.conf` file is not present.. No need for the version, if the config file is there - use the version from `airy.conf`.